addons/resource.language.en_gb/resources/strings.po

```diff
@@ -21110,7 +21113,28 @@ msgctxt "#36637"
 msgid "Note that add-ons installed from zip (excluding served repositories) will not auto-update and must be manually updated. Would you like to proceed?"
 msgstr ""
 
-#empty strings from id 36638 to 36898
+#. Description of setting with label #24163 "Remove orphaned dependencies recursively"
+#: system/settings/settings.xml
+msgctxt "#36638"
+msgid "Remove all modules and support libraries that have been automatically installed as a dependency to other add-ons and are in an orphaned state."
+msgstr ""
+
+#: xbmc/addons/AddonSystemSettings.cpp
```

Comment:
Please add a meaningful "#."  description - here and for the other new strings.

Reply:
done